CallAutomation_OutboundCalling/CallAutomation_OutboundCalling/Program.cs

```diff
@@ -20,7 +20,7 @@
 var targetPhonenumber = "<TARGET_PHONE_NUMBER>";
 
 // Base url of the app
-var callbackUriHost = Environment.GetEnvironmentVariable("VS_TUNNEL_URL")?.TrimEnd('/');
+var callbackUriHost = "<CALLBACK_URI_HOST_WITH_PROTOCOL>";
```

Comment:
One option here would be to reference the IConfiguration provider to get the host URI from configuration or use the tunnel URL depending on how the sample is configured. I think the intent is for us to use Azure Dev Tunnels so we could do something like:

`builder.Configuration["VS_TUNNEL_URL"] ?? builder.Configuration["callbackUri"]`